target_benchmark/dataset_loaders/utils.py

```diff
@@ -157,11 +171,7 @@ def convert_nested_list_to(
         return array_of_arrays_to_df(nested_list)
 
 
-def get_dummy_table_of_format(
-    expected_format: Literal[
-        "array", "nested array", "pandas", "dataframe"
-    ] = "nested array"
-):
+def get_dummy_table_of_format(expected_format: Literal["array", "nested array", "pandas", "dataframe"] = "nested array"):
```

Comment:
What is the diff between "pandas" and "dataframe"?

Reply:
no difference, i think this code is from a while ago and i wrote both options as synonymous to converting to dataframe